src/index.html

```diff
@@ -1,34 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Daily Prebuilt with Kahoot</title>
-  <link rel="stylesheet" href="./style.css" />
+  <title>Daily Prebuilt + Kahoot = <3</title>
+  <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
   <main class="app">
-    <h1>Daily Prebuilt with Kahoot Game Integration</h1>
+    <h1>Daily Prebuilt Kahoot Integration Demo</h1>
     <div class="welcome-screen" id="welcomeScreen">
       <p>
-        Start a Kahoot Game from the apps listed in the embedded video chat
-        widget. Add your daily room url to script.js and click 'Join room'.
+        Start a Kahoot game from the apps listed in the embedded video chat
+        widget. Add your Daily room URL to script.js and click 'Join room'.
       </p>
       <p>
-        You can create a room URL from the
-        <a href="dashboard.daily.co">Daily dashboard</a>.
+        You can create a Daily room and get its URL in the
+        <a href="https://dashboard.daily.co">Daily dashboard</a>.
       </p>
       <button class="welcome-screen-button" id="joinButton">Join room</button>
     </div>
-    <div class="widget-container hidden" id="widgetContainer">
+    <div class="widget-container hidden" id="callContainer">
```

Comment:
I think this naming might be a little more intuitive/self-explanatory to someone reading this demo. I.e., this way they know immediately "ok this sounds like the thing that will contain the call" (widget is a little generic)